CHANGELOG.next.asciidoc

```diff
@@ -15,6 +15,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Update to ECS 1.0.1. {pull}12284[12284] {pull}12317[12317]
 - Default of output.kafka.metadata.full is set to false by now. This reduced the amount of metadata to be queried from a kafka cluster. {pull}12738[12738]
 - Fixed a crash under Windows when fetching processes information. {pull}12833[12833]
+- Update to Golang 1.12.7. {pull}12931[12931]
```

Comment:
For the future.

The Changelog.next.asciidoc is for users only. If we fix important bugs by upgrading go, then we should add Fix xyz in this file. But the fact that the toolchain has been updated is maybe better put in Changelog-developer.next.asciidoc. WDYT?